src/grt/test/macro_obs_not_aligned.ok

```diff
@@ -51,10 +51,10 @@ li1                  0             0            0.00%             0 /  0 /  0
 met1           1173231           178            0.02%             0 /  0 /  0
 met2            784901           250            0.03%             0 /  0 /  0
 met3            586273             0            0.00%             0 /  0 /  0
-met4            199270             0            0.00%             0 /  0 /  0
-met5               422             0            0.00%             0 /  0 /  0
+met4            199550             0            0.00%             0 /  0 /  0
+met5            215220             0            0.00%             0 /  0 /  0
 ---------------------------------------------------------------------------------------
```

Comment:
big jump on met5

Reply:
Similar to the above comment, I expect such a jump because now we have the 1-track minimum, and we have lots of GCells in this design.

Reply:
Similar to the above comment, I expect such a jump because now we have the 1-track minimum, and we have lots of GCells in this design.